### PR TITLE
Add celery task runner and session storage

### PR DESCRIPTION
**Pull request by enku-io**

### CODE REVIEW

File: Dockerfile-base [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+RUN \
+    apt-get install -y dh-autoreconf && \
+    autoreconf -vif && \
+    ./configure  && \
+    make && \
+    make install
+
+
+WORKDIR $HOME
+ENV SNET_DAEMON_V 0.1.6
```
Comment (Habush) [inline Dockerfile-base]: Change the version of the snet daemon to 0.1.8

File: Dockerfile-base [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+RUN mkdir snet-daemon-v$SNET_DAEMON_V
+RUN wget https://github.com/singnet/snet-daemon/releases/download/v$SNET_DAEMON_V/snet-daemon-v$SNET_DAEMON_V-linux-amd64.tar.gz
+RUN tar -xzf snet-daemon-v$SNET_DAEMON_V-linux-amd64.tar.gz -C snet-daemon-v$SNET_DAEMON_V --strip-components 1
+RUN ln snet-daemon-v$SNET_DAEMON_V/snetd snetd
+RUN rm snet-daemon-v$SNET_DAEMON_V-linux-amd64.tar.gz
+
+
+#setup grpc proxy
```
Comment (Habush) [inline Dockerfile-base]: We need to use the latest version of gRPC to add some options like write time outs. Check out the Dockerfile [here](https://github.com/MOZI-AI/moses-service/blob/312462fadcba1c6827208b8331b97066e8794eca/Dockerfile#L32) and change it to something similar

File: service/annotation_server.py [hunk 3]
```diff
@@ -41,25 +62,50 @@ def Annotate(self, request, context):
...
-            return annotation_pb2.AnnotationResponse(graph=response, scm=scm_file)
-
         except Exception as ex:
             logger.error("Error: " + str(ex.__traceback__))
             context.set_code(grpc.StatusCode.INVALID_ARGUMENT)
             context.set_details("Error occurred in while trying to perform request: " + ex.__str__())
-            return annotation_pb2.AnnotationResponse(graph="", scm="")
+            return annotation_pb2.AnnotationResponse(result="url")
+
+        # try:
```
Comment (Habush) [inline service/annotation_server.py]: Why isn't this commented code not removed if you don't use it?

File: Dockerfile-scheme [hunk 4]
```diff
@@ -0,0 +1,32 @@
+FROM opencog/opencog-dev:cli
+MAINTAINER Enkusellasie Wendwosen <enku@singularitynet.io>
+
+#Run apt-get in NONINTERACTIVE mode
+ENV DEBIAN_FRONTEND noninteractive
+
+ENV HOME /root
+WORKDIR $HOME
+
+RUN git clone https://github.com/enku-io/agi-bio.git
```
Comment (Habush) [inline Dockerfile-scheme]: Change this to point to the main repo on opencog or singnet. Or is there a specific reason for this?

File: requirements.txt [hunk 5]
```diff
@@ -3,4 +3,11 @@ grpcio-tools
 pyyaml
 codecov
 pandas
-circus
\ No newline at end of file
+circus
+celery
+python-socketio
```
Comment (Habush) [inline requirements.txt]: where is this dependency used?

File: Dockerfile-scheme (hunk 4 above)
Comment (enku-io) [reply Dockerfile-scheme]: Currently this is the only one that's building.

File: requirements.txt (hunk 5 above)
Comment (enku-io) [reply requirements.txt]: socket-io?

Comment (Habush) [reply requirements.txt]: Yes

**Pull request merged**